operator/v1/types_csi_cluster_driver.go

```diff
@@ -248,6 +252,15 @@ type GCPCSIDriverConfigSpec struct {
 	KMSKey *GCPKMSKeyReference `json:"kmsKey,omitempty"`
 }
 
+// IBMCloudCSIDriverConfigSpec defines the properties that can be configured for the IBM Cloud CSI driver.
+type IBMCloudCSIDriverConfigSpec struct {
+	// encryptionKeyCRN is the IBM Cloud CRN of the customer-managed root key to use
+	// for disk encryption of volumes for the default storage class(es).
+	// +kubebuilder:validation:Pattern:=`^crn:v[0-9]+:bluemix:(public|private):(kms|hs-crypto):[a-z-]+:a/[0-9a-f]+:[0-9a-f-]{36}:key:[0-9a-f-]{36}$`
+	// +optional
+	EncryptionKeyCRN string `json:"encryptionKeyCRN,omitempty"`
+}
```

Comment:
Sharing an example CRN here, because I wasn't familiar with the format:
```
crn:v1:bluemix:public:kms:us-south:a/18240e5ed3f647cb96ca52dc92d9addd:d5e11420-3f0f-447a-92c0-13c978cf9096:key:c4a72f5b-9412-4ce8-80a4-2acb1987f3ba
```
The validation regex looks reasonable to me for this example, but is it valid to reference a key alias in this field? What does a key alias CRN look like, and should this field allow it?